playground-python.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: 'Expanding the Xata Playground: Python Joins TypeScript and SQL'
+description: 'We`ve expanded the Xata Playground to include Python. Now you can write code in Xata`s Playground using Python, TypeScript, and SQL.'
+image:
+  src: https://raw.githubusercontent.com/xataio/mdx-blog/py-play/images/playground-v1.png
+  alt: Xata Playground
+author: Alexis Rico
+date: 09-19-2023
+published: false
+slug: playground-python
+---
+
+At Xata, we're committed to providing you with a versatile and powerful UI to interact with your data. That's why early on we built the [Xata Playground](https://xata.io/playground), a web-based IDE that allows you to write code in TypeScript and SQL to query your data in Xata.
+
+Today, we're excited to announce that we've expanded the Xata Playground to include Python. Now you can write code in Xata's Playground using Python, TypeScript, and SQL.
+
+In this post, we'll do a technical deep dive into how the Xata Playground works and how we added Python to the mix.
+
+## The Xata Playground
+
+As a new hire that had just marked my first month at Xata, the 8th of April 2022, I opened a pull request with the first version of the Xata Playground. It was a very simple proof of concept that I wanted to show during our weekly team meeting.
+
+It was a simple [monaco-editor](https://microsoft.github.io/monaco-editor/) where you could write TypeScript code and run it to see the results in a separate panel. The main goal was that our users could try out our TypeScript SDK without having to install anything.
+
+Both internally and with some early adopters, we noticed that it was something we had to invest in, and we decided to iterate quickly over the proof of concept and release it as soon as possible. On the 2nd of May 2022, we released the first version of the Xata Playground.
```

Comment:
```suggestion
Both internally and with some early adopters, we noticed that it was something we had to invest in, and decided to iterate quickly over the proof of concept and release it as soon as possible. A month later, we released the first version of the Xata Playground. And it's still one of our most popular features.
```

The exact date felt a bit unnecessary and I was thinking we could call out how popular it is.